fdm-calculator/src/norms/nl/2025/stikstofgebruiksnorm.ts

```
@@ -169,15 +146,25 @@ export async function getRegion(
  */
 function getNormsForCultivation(
     selectedStandard: NitrogenStandard,
-    b_lu_variety: string | undefined,
-    is_derogatie_bedrijf: boolean | undefined,
     b_lu_end: Date,
+    subTypeOmschrijving?: string, // New parameter
 ): NormsByRegion | undefined {
-    // Handle sub-types, typically used for temporary grasslands where norms depend on the period.
     if (selectedStandard.sub_types) {
+        let matchingSubType
```

Biome (2.1.2)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)